src/MongoDB/Exception/WriteException.c

```diff
@@ -57,8 +57,11 @@ PHP_METHOD(WriteException, getWriteResult)
 		return;
 	}
 
-
+#if PHP_VERSION_ID >= 70000
+        zend_read_property(php_phongo_writeexception_ce, getThis(), ZEND_STRL("writeResult"), 0 TSRMLS_CC, writeresult);
```

Comment:
The TSRMLS_CC can be removed here, but more importantly is this right?
What does zend_read_property() return now?